### ISSUE TITLE
TypeError: expected np.ndarray (got numpy.ndarray)

### ISSUE THREAD

**Issue by suws0501** [sgrvinod/a-PyTorch-Tutorial-to-Object-Detection#103]
I download the dataset and the model according to the tutorial but an error like "TypeError: expected np.ndarray (got numpy.ndarray)" plagued me the whole afternoon. I tried searching on google and got answers like "downgrade version numpy < 1.26.4, downgrade torchvision" but nothing worked. However, in the frustration, I wandered around the internet and found a post about Python code stop working after a few years due to back-compatability issues, so I got back and downgrade my Python version to 3.6 and IT WORKED. 
So basically I got the code to run while writing this post, but I still fell a compulsion to warn newcomers on the issue: always downgrading your Python version to 3.6 !! Pytorch is confusing enough without this version problems and now urrghh

**Comment by azhanisyhptr** [comment]
thank you for the advice. i'll stick to it !
